lib/common/utils.ts

```diff
@@ -60,6 +60,7 @@ export const isMix: boolean = typeof process !== 'undefined' &&
     !!(typeof window !== 'undefined' && (window as any)['HTMLElement']);
 
 export function patchProperty(obj: any, prop: string) {
+  // TODO: @JiaLiPassion, if the desc not exists in obj, should we still patch the property?
```

Comment:
I don't think so. 

The way I imagine this to work is that we have a list of all properties, and than we only patch those properties, but only if the property already exist. But we don't look for extra ones.